src/components/navbars/topbar/api/pickApi.tsx

```diff
@@ -176,7 +176,6 @@ export default function PickApi() {
           copyStatus.azureConvoStatus     = STATUS.AVAILABLE;
           copyStatus.whisperStatus        = STATUS.AVAILABLE;
           copyStatus.streamTextStatus     = STATUS.AVAILABLE;
-          copyStatus.scribearServerStatus = STATUS.NULL;

```

Comment:
It would be very helpful to explain WHY this is special ie. we want to keep this unchanged (whereas the others are reset)